modules/riemann/riemann-grammar.ym

```diff
@@ -140,6 +140,10 @@ riemann_option
           {
             log_threaded_dest_driver_set_flush_lines(last_driver,  $3);
           }
+        | KW_FLUSH_TIMEOUT '(' nonnegative_integer ')'
```

Comment:
Would it be possible to move this to the `lib/cfg-grammar.y` something like options for threaded dest drivers ?

Reply:
IMHO This is resolved by #2386

Reply:
I have now rebased this branch, and indeed flush-*() related options are now handled by a rule
in cfg-grammar, which is just referenced from here.

This resolves this issue.